### PR TITLE
Fix Ellipsize urls

### PR DESCRIPTION
**Pull request by enukeabhi**
#70

### CODE REVIEW

File: src/constants.js [hunk 1]
```diff
@@ -124,3 +124,6 @@ export const GESTURE_SEGMENT_HINT_TIMEOUT = 500
 
 // number of recently edited thoughts to store
 export const RECENTLY_EDITED_THOUGHTS_LIMIT = 100
+
+// number of chars to show for a url
+export const ELLIPSIS_URL_LENGTH = 40
```
Comment (raineorshine) [inline src/constants.js]: Thanks for putting this in the constants file!

Comment (raineorshine) [reply src/constants.js]: Let's call this URL_MAX_CHARS

File: src/util/ellipsizeUrl.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import {
+    ELLIPSIS_URL_LENGTH,
+  } from '../constants.js'
+import { isURL } from '../util/isURL'
+
+export const ellipsizeUrl = value => {
+    if (value && isURL(value)) {
+        return value.substring(0, ELLIPSIS_URL_LENGTH) + '...'
+    }
+    return value
```
Comment (raineorshine) [inline src/util/ellipsizeUrl.js]: Let's use a ternary instead of an `if` statement here.

File: src/constants.js [hunk 3]
```diff
@@ -124,3 +124,7 @@ export const GESTURE_SEGMENT_HINT_TIMEOUT = 500
 
 // number of recently edited thoughts to store
 export const RECENTLY_EDITED_THOUGHTS_LIMIT = 100
+
+// number of chars to show for a url
+// export const ELLIPSIS_URL_LENGTH = 40
```
Comment (raineorshine) [inline src/constants.js]: Remove old constant

File: src/util/ellipsizeUrl.js [hunk 4]
```diff
@@ -0,0 +1,8 @@
+import {
+    URL_MAX_CHARS,
+  } from '../constants.js'
+import { isURL } from '../util/isURL'
+
+export const ellipsizeUrl = value => {
+    return (value && isURL) ? value.substring(0, URL_MAX_CHARS) + '...' : value
```
Comment (raineorshine) [inline src/util/ellipsizeUrl.js]: Parentheses are redundant since order-of-operations of ternary is lower than `+`.

**Comment by raineorshine** [comment]
Looks good! Please rebase on dev so I can merge.

**Comment by enukeabhi** [comment]
> Looks good! Please rebase on dev so I can merge.

I have rebased with dev. Please merge.

**Pull request merged**